### PR TITLE
Add carbon-crypto-service based operators for JWT crypto operations.

### PR DESCRIPTION
**Pull request by karu95**
## Proposed changes in this pull request
This PR contains the improvements that are required to use Carbon Crypto Service for JWT token encryption, decryption, sign and verification. 

## When should this PR be merged
Following PRs should be merged before this.
* https://github.com/wso2/carbon-kernel/pull/1880

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/cryptooperators/ServiceProviderKeyResolver.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            if (StringUtils.isNotBlank(certificateContent)) {
+                // Build the Certificate object from cert content.
+                return new CertificateInfo(String.valueOf(serviceProvider.getApplicationID()),
+                        IdentityUtil.convertPEMEncodedContentToCertificate(certificateContent));
+            } else {
+                log.error("Public certificate not configured for Service Provider with " +
+                        "client_id: " + clientId + " of tenantDomain: " + tenantDomain);
+            }
+        } catch (CertificateException e) {
+            String errorMessage = "Error occurred while converting the PEM encoded service provider certificate.";
```
Comment (karu95) [inline components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/cryptooperators/ServiceProviderKeyResolver.java]: Add client id for error information.

**Comment by malithie** [comment]
@JKAUSHALYA , since this is with related to a feature, can we create a feature branch and get this merged there

**Comment by malithie** [comment]
@JKAUSHALYA , looks like there are some conflicts. Can we get them addressed or else we might have to close this PR

**Comment by madurangasiriwardena** [comment]
@karu95 Please resolve the conflicts and reopen the PR.

**Pull request closed without merging**